HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,125 @@
-
-### Graph 2
-```{r, echo = T, include = TRUE}
-data("iris")
-```
-
-Using the iris dataset, make this graph:
-```{r graph 2 code, echo=FALSE}
-
-```
-
-
-### Graph 3
-You'll need the information in this first box to create the graph
-```{r, echo = TRUE}
-data("mpg")
-corvette <- mpg[mpg$model == "corvette",]
-#install
-require("ggrepel") #useful for making text annotations better, hint hint
-set.seed(42)
-```
-
-Now using the mpg dataset and the corvette dataset, make this graph:
-
-```{r graoh 3 code}
-
-```
-
-There is a trick to getting the model and year to print off together. `paste()` is a useful function for this, also pasting together parts of file names and parts of urls together. 
-
-
-### Graph 4 
-```{r, echo = T}
-data(mpg)
-
-#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
-library(RColorBrewer)
-display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
-```
-
-The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
-
-
-
-Now using the above mpg dataset, make this graph
-
-```{r graph 4 code}
-
-```
-
-
-
-
+---
+title: "HW02_B_Graph-Mimic"
+author: "YOUR NAME HERE"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+data("diamonds")
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+
+ggplot(diamonds)+
+  geom_bar(aes(cut, ..count.., fill = clarity), position = "dodge") + 
+  labs(title = "My Diamond Collection", x = "Diamond Cut", y = "Number of Diamonds")+
+  theme(plot.title = element_text(hjust = 0.5))
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+  
+
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+#this is of using the facets
+ggplot(iris, aes(x = Sepal.Length, y = Petal.Length, color = Species, shape = Species)) + 
+  geom_point()+
+  geom_smooth(method = lm)+
+  facet_wrap(factor(iris$Species, levels = c("versicolor","setosa","virginica")))
+```
```

Comment:
Another nice plot. You were able to get the correct order of species from left to right, with the same colors, shapes, and linear trendline.
The only things you missed was scaling the y-axis with 'scales = "free_y" ' and a little bit more formatting with your linear fit.

Reply:
Thank you learnt a lot!